ExecutionHelper/deploy/deploy.ts

```diff
@@ -1,9 +1,13 @@
 import { deployContract } from "./utils";
+import * as fs from 'fs';
 
-// An example of a basic deploy script
-// It will deploy a Greeter contract to selected network
-// as well as verify it on Block Explorer if possible for the network
-export default async function () {
+function storeContractAddress(address: string) {
+  fs.writeFileSync(".executionHelper.address", address);
```

Comment:
In the future we could write a JSON, since JS has the built-in capabilities for it, but this simple approach works for now.

Reply:
Yeah, There is a lot of room for improvement (look at my comment about how it might fail) but I think that duct-taping it is the best solution for now.